cpp/src/decisiontree/batched-levelalgo/metrics.cuh

```diff
@@ -76,7 +76,8 @@ DI void giniGain(int* shist, DataT* sbins, Split<DataT, IdxT>& sp, IdxT col,
   for (IdxT i = threadIdx.x; i < nbins; i += blockDim.x) {
     int nLeft = 0;
     for (IdxT j = 0; j < nclasses; ++j) {
-      nLeft += shist[i * 2 * nclasses + j];
+      //nLeft += shist[i * 2 * nclasses + j];
```

Comment:
nitpick: please remove the commented code. (applies all other such comments in this file)

Reply:
😅 done